python/ray/worker.py

```diff
@@ -236,6 +236,18 @@ def __init__(self):
         # CUDA_VISIBLE_DEVICES environment variable.
         self.original_gpu_ids = ray.utils.get_cuda_visible_devices()
         self.profiler = profiling.Profiler(self)
+        # A dictionary that maps from driver id to SerializationContext
+        # TODO: clean up the SerializationContext once the job finished.
+        self.serialization_context_map = dict()
+        # Identity of the driver that this worker is processing.
+        self.task_driver_id = None
+
+    def get_serialization_context(self, driver_hash):
+        """Get the SerializationContext of the driver that this worker is processing
+        """
```

Comment:
Why pass in `driver_hash` instead of just using `self.task_driver_id`?

Reply:
The design is for register_class_for_serialization. While calling it through run_function_on_all_workers, the function will be serialized by pickle. However pickle cannot serialize class ObjectId, so I use a hash instead of task_driver_id, then hash is used in the map to make the behavior consistent.